File: a/Ram.cpp
class Ram
{
	private:
		int capacity;
		int price;

	public:
		// construct
		Ram(int capacity, int price)
		{
			this->capacity = capacity;
			this->price = price;
		}

		//default construct
		Ram(){}

		void setCapacity(int capacity) { this->capacity = capacity; }
		void setPrice(int price) { this->price = price; }

		int getCapacity() { return this->capacity; }
		int getPrice() { return this->price; }

		void printRam()
		{
			cout << "Kapasitas RAM : " << this->capacity << endl;
			cout << "Harga RAM : " << this->price << endl;
		}

		// deconstruct
		~Ram(){}
};

File: b/Ram.py
class Ram():
	__capacity: int
	__price: int

	def __init__(self, capacity:int, price:int):
		self.__capacity = capacity
		self.__price = price

	def setCapacity(self, capacity:int): self.__capacity = capacity
	def setPrice(self, Price:int): self.__price = Price

	def getCapacity(self): return self.__capacity
	def getPrice(self): return self.__price

	def printRam(self):
		print("Kapasitas RAM : ", self.__capacity)
		print("Harga RAM : ", self.__price)